client/state/data-layer/wpcom-http/test/index.js

```diff
@@ -77,3 +80,99 @@ describe( '#queueRequest', () => {
 		}, 10 );
 	} );
 } );
+
+describe( '#fetcherMap', () => {
+	const wpcomReq = {};
+	const noopCallback = () => {};
+
+	describe( 'GET', () => {
+		let getFooAction;
+
+		beforeEach( () => {
+			wpcomReq.get = spy();
+			getFooAction = {
+				method: 'GET',
+				path: '/foo',
+			};
+		} );
+
+		it( 'should send with query', () => {
+			getFooAction.query = {
+				apiVersion: '2.0',
+				apiNamespace: 'wp/v2',
+			};
```

Comment:
I honestly don't feel strongly about either :) I went that way (`let` + new `spy`-s in `beforeEach` to match what was already there for `dispatch` / `next`). If we go with `const` + `resets`, it's probably worth updating the entire file.